### PR TITLE
Issue #700

### PR DESCRIPTION
**Pull request by sjaakluthart**

### CODE REVIEW

File: src/date-separator/styles.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const styles = {
+  root: {
+    position: 'relative',
+    margin: '16px 0',
+    width: '100%',
+    display: 'flex',
+    alignItems: 'center'
+  },
+  text: {
+    textAlign: 'center',
```
Comment (IanCStewart) [inline src/date-separator/styles.js]: I think this is redundant... If width is automatic text will be in the center from the start

File: src/date-separator/styles.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    fontSize: '16px',
+    fontFamily: 'inherit',
+    padding: '0 8px'
+  },
+  hr: {
+    height: '1px',
+    border: 0,
+    margin: 0,
+    backgroundColor: colors.grey,
+    flex: 1
```
Comment (IanCStewart) [inline src/date-separator/styles.js]: When using this on `hr` but not on `text` make sure text can't shrink. See message styling for this fix

File: src/message/styles.js [hunk 3]
```diff
@@ -1,4 +1,7 @@
 export default {
+  root: {
```
Comment (IanCStewart) [inline src/message/styles.js]: Could you explain why we need this?

Comment (sjaakluthart) [inline src/message/styles.js]: Position relative is redundant, will remove

**Pull request merged**